### ISSUE TITLE
High CPU usage on versions newer than 0.4.8

### ISSUE THREAD

**Issue by amarce** [Achiefs/fim#149]
Hi!

I'm testing FIM to install it into our production servers, and we are seeing a 100% cpu usage on one core using 0.4.9 and 0.4.10 versions, on 0.4.8 is not using any CPU at all. It even uses 100% of CPU with only monitoring /sbin and nothing else.

Thanks!

**Comment by okynos** [comment]
Hello @amarce,

Could you attach here, your `config.yml` and your `fim.log` file in debug mode?
In addition which operating system are you using? 

We will try to reproduce it. Thanks for your feedback.

**Comment by okynos** [comment]
@amarce We found a problem into rotator thread. Could you test the following packages? Tell us if the problem is solved

[fim-0.4.11_deb_rpm.zip](https://github.com/Achiefs/fim/files/15030966/fim-0.4.11_deb_rpm.zip)

We will release it as soon as all tests and checks has been passed.

**Comment by amarce** [comment]
Hi,

This release is working perfectly. Thanks a lot for the fix. Regarding the rotation, does it have any possible configuration on config file or it gets rotated magically? I setted up a logrotate policy, but if it can be done natively would be great.

**Comment by okynos** [comment]
Hi @amarce,

Thanks for your feedback we are working to release this fix asap.
Yes, FIM includes a policy to manage events and log rotation. By default it is set to 128 MB and FIM compress the rotated file.
You could change this value using these parameters:
```
node: "FIM"

# Events configuration, where to store produced events
events:
  max_file_size: 128

# App procedure and errors logging
log:
  max_file_size: 128
```
The main problem is the timer. Currently, It is not configurable. With this fix, We will set it up to 30 minutes on each status check.
We will include an issue to configure this timer with parameter.

**Comment by amarce** [comment]
Nice, do we have a max number of files too or should they be deleted manually?

Thanks again!

**Comment by okynos** [comment]
It should be deleted manually, but it is a good feature, will create an issue with that. Thanks.

**Comment by okynos** [comment]
Hi @amarce,

We just released v0.4.11 fixing this issue. Due the main branch is up in the development than 0.4 we will need to resolve conflicts. Meanwhile the 0.4.11 code is fully tested and ready, take a look at https://github.com/Achiefs/fim/releases/tag/v0.4.11

This issue will be closed after the pulll request merge, don't hesitate to reopen it at your consideration.
